Scripts/backtotop.js

```diff
@@ -0,0 +1,17 @@
+let button = document.getElementById("topButton");
+
+window.onscroll = function() {scrollFunction()};
```

Comment:
Scrie event-urile așa cum v-am arătat

```suggestion
window.addEventListener('scroll', function() {scrollFunction()});
```

Chestiile asta cu `.on...` sunt un sistem mai vechi de a înregistra un event.